docs/user/guides/best-practice/index.rst

```diff
@@ -0,0 +1,36 @@
+How-to guides: best practice
+============================
+
+Over the years,
+we have become familiar with a number of methods that work well and which we consider **best practice**.
+
+⏩️ :doc:`Best practices for linking to your documentation </automatic-redirects>`
+    Documentation changes over time,
+    and links and cross-references can become challenging manage for various reasons.
+    Here is a set of best practices explaining and addressing these challenges.
+
+⏩️ :doc:`Deprecating content </guides/deprecating-content>`
+    Best practice for removing or deprecating documentation content.
+
+⏩️ :doc:`Create reproducible builds </guides/reproducible-builds>`
+    Using Sphinx, themes and extensions means that your documentation project needs to fetch a set of dependencies,
+    each with a special version.
+    Over time, using an unspecified version means that documentation projects suddenly start breaking.
+    In this guide,
+    you learn how to secure your project against sudden breakage.
+    **This is one of our most popular guides!**
+
+⏩️ :doc:`Search Engine Optimization (SEO) for documentation projects </guides/technical-docs-seo-guide>`
+    This article explains how documentation can be optimized to appear in search results,
+    ultimately increasing traffic to your docs.
```

Comment:
```suggestion
    increasing traffic to your docs.
```

Reply:
ultimately, you always remove that word :smile:  :+1: